pyproject.toml

```diff
@@ -54,13 +54,17 @@ MCEq = {version = "*", optional = true}
 crflux = {version = "*", optional = true}
 mattak = {git = "https://github.com/RNO-G/mattak", optional = true}
 rnog-runtable = {git = "ssh://git@github.com/RNO-G/rnog-runtable.git", optional = true}
+pre-commit = {version = "*", optional = true}
```

Comment:
can't we make this a required dependency?

Reply:
It's not a required dependency for users, only for developers.

Reply:
what does that mean? Is it a problem if user install it?

Reply:
It's not a problem but also doesn't do anything (in fact, it does nothing until you run `pre-commit install` anyway), so I don't see the benefit of adding this as a dependency for everyone.